### ISSUE TITLE
Deep Multi-scale Convolutional Neural Network for Dynamic Scene Deblurring 

### ISSUE THREAD

**Issue by jskim19** [GDSC-Ewha-4th/Study-paperReview#50]
### Abstract
카메라의 움직임이나 물체의 모션으로 인한 블러는 컴퓨터 비전에서 큰 challenge였다. 이전까지의 blur remove 방법들은 대부분 blur kernel이 uniform하거나 locally linear하다고 가정하고 진행했는데, blur kernel이 approximation하기 어려운 경우 실패했다. 이 논문에서는 multi-scale CNN을 통해 이미지를 복원한다. 
 
### Introduction 
카메라가 흔들리거나 물체가 흔들리면 이미지의 퀄리티가 저하된다. 블러를 제거하기 위한 방법들은 보편적으로
B=KS+n의 모델을 따르는데, 
(B: blurry image, K: large sparse matrix contain a local blur kernel acting on S, S: latent sharp image, n: noise)
실제로는 블러 커널이 unknown이기 때문에 latent sharp image S, blur kernel K를 함께 estimation한다. 

몇몇 논문에서는 3d 카메라의 모션으로만 블러가 생긴다고 가정하지만 실제로는 오브젝트도 움직인다. 이에 따라 다양한 방법이 제시되었지만 blur kernel approximation이 여전히 아직 정확하지 않다는 단점이 존재한다. 또한 최근에 몇몇 specific한 타입에는 잘 맞는 CNN 적용이 사용되기도 했지만 이 역시도 common하게 적용하기에는 제약이 있다. 존재하는 모든 방법은 simple & unrealistic blur kernel models을 사용함으로 인한 문제점들이 있다. 따라서 이 논문에서는 다른 방법을 제시한다. 

- Multi-scale CNN : directly restore latent images w/o assuming any restricted blur kernel model. explicit한 블러 커널을 찾지 않기 때문에 kernel estimation error에서 자유롭다. 

- Multi-scale loss로 training. converge에 도움이 된다. 

- New realistic blurry image dataset w/ ground truth sharp image 제안 

보통은 blur kernel을 찾는 것이 일반적이다. 하지만 아래와 같은 단점들이 존재한다. 

- Occluded region / depth variation에는 적용하기 어려움 

- Noise & saturation에 subtle하고 sensitive하다. 

- 잘못된 estimation은 image에 인공적인 변화를 가져다줌. 

따라서 이 논문에서는 Kernel-free method를 사용한다. 

### Blur Dataset 
kernel 을 모델링하는 방법 대신 sharp information을 시간에 따라 integration 시키는 방법을 선택했다. 적분된 시그널은 nonlinear CRF를 통해 pixel 값으로 변환된다. 
카메라 센서는 노출 시간동안 빛을 받아들이고, 이미지는 매 시간 축적되며 blurry image를 만든다. 따라서 적분을 통해 이 과정을 approximation 한다. 

### Proposed Method 
![image](https://user-images.githubusercontent.com/113578083/201513400-bdb8c5f4-395f-48e7-8120-39ca1075c8a3.png)
a) Original residual network building block
b) Modified building block of our network 

- batch normalization layer 사용 X, convergence speed 높이기 위해 ReLU 제거
- Coarset level network : Unsampling 대신 upconvolutoin 사용 -> better performance 
- Finer level network : coarset level과 basically same하지만 첫번째 convolution layer가 이전 단계에서의 sharp feature를 가짐

### Conclusion 
kernel estimation으로 인한 문제들을 막을 수 있고, 이 접근으로 더 빠른 속도를 갖추며 qualitatively and quantitatively outperform했다.

**Comment by Soyeon-Cha** [comment]
정리를 잘해주셔서 이 논문이 제시하는 방식이 기존 방식과 어떻게 다른지에 집중해서 흥미롭게 읽을 수 있었습니다!
리뷰 감사합니다 :)
